File: Crypto-Trader-Library/Api-Library/Api-Models/src/main/kotlin/org/cryptotrader/api/library/entity/user/SubscriptionTier.kt
package org.cryptotrader.api.library.entity.user

enum class SubscriptionTier(val label: String, val level: Int, val intervalMs: Long) {
    FREE("Free", 1, 30000L),
    PRO("Pro", 2,  1000L),
    ULTIMATE("Ultimate", 3, 5000L);

    companion object {
        fun fromLabel(label: String): SubscriptionTier? {
            return entries.find { it.label == label }
        }
    }
}

File: Crypto-Trader-Website/angular/components/elements/element-group-promo/tier-promo/models/SubscriptionTier.ts
export enum SubscriptionTier {
    Free = 'Free Tier',
    Pro = 'Pro Tier',
    Ultimate = 'Ultimate Tier',
}
export namespace SubscriptionTier {
    export function values(): SubscriptionTier[] {
        return Object.values(SubscriptionTier) as SubscriptionTier[]
    }
}
